go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
 	github.com/golang/snappy v0.0.3
 	github.com/google/go-cmp v0.4.0
+	github.com/google/uuid v1.6.0
```

Comment:
Please don't bring new dependancy without real necessity, in this case it is not realy needed.

Reply:
Fixed